package.json

```diff
@@ -17,7 +17,9 @@
   },
   "dependencies": {
     "@types/lodash": "^4.14.108",
+    "@types/mz": "0.0.32",
     "mkfifo": "^0.1.5",
+    "mz": "^2.7.0",
```

Comment:
NIT: this should have been in the next commit.

Reply:
It was actually used in the previous commit, just all of the usages got moved out to the config-backend module after the refactor.